nbstripout/_nbstripout.py

```diff
@@ -287,6 +298,10 @@ def main():
                       help='Print version')
     parser.add_argument('--force', '-f', action='store_true',
                         help='Strip output also from files with non ipynb extension')
+    parser.add_argument('--mode'
```

Comment:
Set the default to `jupyter`. Also please format according to PEP 8

Reply:
done